service/scaladsl/api/src/main/scala/com/lightbend/lagom/scaladsl/api/transport/Exceptions.scala

```diff
@@ -153,13 +153,18 @@ object TransportErrorCode {
     }
   }
 
-  private case class TransportErrorCodeImpl(http: Int, webSocket: Int, description: String) extends TransportErrorCode
+  private case class TransportErrorCodeImpl(http: Int, webSocket: Int, description: String) extends TransportErrorCode {
+    override def toString = s"$http/$webSocket $description"
+  }
```

Comment:
Currently when you `TransportException.toString`, you get this:

```
com.lightbend.lagom.scaladsl.api.transport.DeserializationException: client tim already exists (TransportErrorCodeImpl(400,1003,Unsupported Data/Bad Request))
```

This changes it so that you get:

```
com.lightbend.lagom.scaladsl.api.transport.DeserializationException: client tim already exists (400/1003 Unsupported Data/Bad Request)
```